tests/remote_attestation/maa/sgxlkl_enclave/makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: all build clean alpine-container alpine-binaries lkl-image lkl-run
+
+COMMONS="../common"
+SGXLKL_BIN_ROOT="$(SGXLKL_PREFIX)/bin"
+BUILD_DIR="/src/sgxlkl_enclave"
+SGXLKLAPP_COMMON="/src/common"
+
+build-alpine-exe = sudo gcc -g -o $(BUILD_DIR)/alpine_sgxLklApp -fshort-wchar $(BUILD_DIR)/sgxLklApp.c $(BUILD_DIR)/clientAgent_init.c $(BUILD_DIR)/tlscli.c $(SGXLKLAPP_COMMON)/log_utils.c $(SGXLKLAPP_COMMON)/auth.cpp $(SGXLKLAPP_COMMON)/maa.cpp $(SGXLKLAPP_COMMON)/transport_utils.c $(SGXLKLAPP_COMMON)/curl_helper.cpp $(SGXLKLAPP_COMMON)/json_utils.c $(BUILD_DIR)/oe_private/verify_cert.c $(BUILD_DIR)/oe_private/cert.c -lmbedtls -lmbedx509 -lmbedcrypto -ldl -lcurl
+
+
+all: lkl-image
+
+clean:
+	rm -f clientAgent.o alpine_sgxLklApp sgxlkl-image.img*
+
+alpine-container:
+	docker build -t alpine3.10.build ./alpine
+
+alpine-binaries: alpine-container
+	docker run --rm  -v $(CURDIR)/..:/src alpine3.10.build bash -c "$(build-alpine-exe)"
```

Comment:
we can run `$(build-alpine-exe)` in step `lkl-image`, instead of creating a separate docker container for building the executable. This way we can get rid of `alpine-container`, `alpine-binaries`, and the Dockerfile under `alpine`.